ops/docker/Dockerfile.geth

```diff
@@ -12,6 +12,6 @@ FROM alpine:latest
 RUN apk add --no-cache ca-certificates jq curl
 COPY --from=builder /go-ethereum/build/bin/geth /usr/local/bin/
 
-EXPOSE 8545 8546 8547
+EXPOSE 8545 8546 8547 8548
```

Comment:
We shouldn't add another port here - adding the port to the docker compose file should be fine